.github/workflows/CI.yml

```
@@ -70,7 +70,9 @@ jobs:
           args: -i python --release --out dist --no-sdist
       - name: Install built wheel
```

Try bash? https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell

```suggestion
      - name: Install built wheel
        shell: bash
```